github_api/voting.py

```diff
@@ -117,19 +117,35 @@ def get_vote_weight(api, username):
     age = (now - created).total_seconds()
     old_enough_to_vote = age >= settings.MIN_VOTER_AGE
     weight = 1.0 if old_enough_to_vote else 0.0
+
+    if is_collaborator:
+        weight *= 1.5
+
+    # sorry smitty, No food for you (http://e.lvme.me/7mb4jy9.jpg)
     if username.lower() == "smittyvb":
         weight /= 2
 
     return weight
 
 
+def get_collaborators(api, urn):
```

Comment:
this might be a good function to cache.  see https://github.com/chaosbot/Chaos/blob/master/patch.py#L29-L32